src/index.ts

```diff
@@ -105,11 +107,11 @@ export class ErrorReporting {
       (err: {}, request?: Request, additionalMessage?: string|{},
        callback?: Callback|{}|string) => ErrorMessage;
   event: () => ErrorMessage;
-  hapi: {register: (server: h.Server, options: {}, next: Function) => void};
-  express: (err: {}, req: e.Request, res: e.Response, next: Function) => void;
-  restify:
-      (client: AuthClient, config: Configuration, server: r.Server) => void;
-  koa: (next: Function) => Iterable<Function>;
+  hapi: {register: (server: {}, options: {}, next: Function) => void};
+  express: (err: {}, req: {}, res: {}, next: Function) => void;
+  restify: (server: {}) => RequestHandler | RequestHandler[];
+  // tslint:disable-next-line:no-any
+  koa: (context: any, next: () => Promise<{}>) => {};
```

Comment:
Why is `context` of type `any`?

Reply:
I tried to make this of type `{}` or `{[key: string]: {}`, etc. but would always get compilation errors.  So I used `any`.

Reply:
When adding a tslint exclusion, it is good to include the reasoning in the comment itself. It would be good to the 'why' a bit better in this case though.